test/fixtures/env.js

```diff
@@ -1,11 +1,11 @@
 module.exports = {
   testEnv: {
     FOREST_URL: 'http://localhost:3001',
-    TOKEN_PATH: '.',
+    TOKEN_PATH: './test/services/tokens',
   },
   testEnv2: {
     FOREST_URL: 'http://localhost:3001',
-    TOKEN_PATH: '.',
+    TOKEN_PATH: './test/services/tokens',
```

Comment:
common path, we replace '.' by a path to not pollute the root of the project.
Note: './test/services/tokens' is in gitignore 👍